command-line-interface.md

```diff
@@ -0,0 +1,246 @@
+# Operations
+
+The [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
The at least one API, that only has one API choice means a runtime MUST provide the following command line interface.  Then inside the command line interface document it clarifies that statement by again declaring that the runtime MUST provide the following. 

If you intended to command line interface to be optional for those implementing a command line interface... that's not what it says.  It has been my understanding that the command line interface specification is only for those runtimes providing a command line interface. What am I missing here?